app/src/main/java/com/koalatea/sedaily/feature/player/AudioService.kt

```diff
@@ -201,7 +201,10 @@ class AudioService : LifecycleService() {
                             .build()
                 }
             })
-
+            // override media connector increment speed
```

Comment:
Can you add a more descriptive comment, or feel free to get rid of

Reply:
Got rid of comment completely